src/components/SideNav/__snapshots__/SideNav.story.storyshot

```diff
@@ -164,13 +166,11 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/SideN
             >
               <li
                 className="bx--side-nav__menu-item"
-                role="none"
               >
                 <button
                   className="bx--side-nav__link"
                   label="Yet another link"
                   onClick={[Function]}
-                  role="menuitem"
```

Comment:
I don't remember why this was originally added. We should double check to see if this and the role=none above change anything

Reply:
It looks like it helps tell the accessibility let it know to shift focus to this item when you tab into the list

Reply:
Yep, to my knowledge this was an intentional fix from Carbon. Neither are something we set.